libs/sdk-ui-kit/src/Header/HeaderUpsellButton.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import { FormattedMessage, injectIntl, IntlShape } from "react-intl";
+
+export interface IUpsellButtonProps {
+    intl: IntlShape;
+    onUpsellButtonClick?: (e: React.MouseEvent) => void;
+}
+
+export const CoreHeaderUpsellButton: React.FC<IUpsellButtonProps> = ({ onUpsellButtonClick }) => {
```

Comment:
Please remove the prop `intl` from `IUpsellButtonProps` and rewrite the interface as:

`export const CoreHeaderUpsellButton: React.FC<IUpsellButtonProps>`

↓

`export const CoreHeaderUpsellButton: React.FC<IUpsellButtonProps & WrappedComponentProps>`

to make it consistent with other components:

- https://github.com/gooddata/gooddata-ui-sdk/blob/e490e01b4d66137e6468f7567365a21b29a153e8/libs/sdk-ui-kit/src/Header/HeaderMenu.tsx#L10
- https://github.com/gooddata/gooddata-ui-sdk/blob/e490e01b4d66137e6468f7567365a21b29a153e8/libs/sdk-ui-kit/src/Header/HeaderAccount.tsx#L11

Reply:
Also, I think we should move the definition of `IUpsellButtonProps` to l[ibs/sdk-ui-kit/src/Header/typings.ts](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/src/Header/typings.ts) for the sake of consistency with interfaces used by other components.

- https://github.com/gooddata/gooddata-ui-sdk/blob/e490e01b4d66137e6468f7567365a21b29a153e8/libs/sdk-ui-kit/src/Header/HeaderMenu.tsx#L8
- https://github.com/gooddata/gooddata-ui-sdk/blob/e490e01b4d66137e6468f7567365a21b29a153e8/libs/sdk-ui-kit/src/Header/HeaderAccount.tsx#L8

Reply:
I changed it thanks